src/eko/evolution_operator/flavors.py

```diff
@@ -160,9 +160,13 @@ def rotate_matching(nf, inverse=False):
             l[f"{oth}.{qpm}"] = -(nf - 1.0)
     # also higher quarks do not care
     for k in range(nf + 1, 6 + 1):
-        q = quark_names[k - 1]
-        for sgn in "+-":
-            l[f"{q}{sgn}.{q}{sgn}"] = 1.0
+        # q = quark_names[k - 1]
+        # for sgn in "+-":
+        #     l[f"{q}{sgn}.{q}{sgn}"] = 1.0
+        n = k ** 2 - 1
+        l[f"V{n}.V"] = 1.0
+        l[f"T{n}.S"] = 1.0
+        l[f"T{n}.g"] = 1.0
```

Comment:
- this seems wrong: we agreed to do the matching in the basis which does include (e.g.) `c+,c-` but NOT `T15,V15`
- instead https://github.com/N3PDF/eko/blob/7716902548c60646ae6340b12bd27b5800a2e0d3/src/eko/matching_conditions/__init__.py#L60 again here only `c+,c-` should appear, both in the input and the output